sql/open_address_hash.h

```diff
@@ -205,9 +242,15 @@ template <typename key_trait, typename trait> class open_address_hash
     }
 
     if (unlikely(capacity > 7 && (_size - 1) * LOW_LOAD_FACTOR < capacity))
-      rehash(0.5 * capacity);
+      shrink(0.5 * capacity);
```

Comment:
if it's `grow(capacity << 1);`, then maybe let's make it symmetric with capacity >> 1?

Reply:
it's done